django_mongodb_backend/management/commands/get_encrypted_fields_map.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import DEFAULT_DB_ALIAS, connections
+
+
+class Command(BaseCommand):
+    help = "Generate an encryptedFieldsMap for MongoDB automatic encryption"
+
+    def handle(self, *args, **options):
+        connection = connections[DEFAULT_DB_ALIAS]
```

Comment:
Like other commands, there should be a `--database` argument.

Reply:
b07c3e6817faf810eec7fa6b8df4e31c03ee3293